File: app/src/main/java/com/dev/ome/corentin/xkcdapp/XkcdService.java
package com.dev.ome.corentin.xkcdapp;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface XkcdService {
    @GET("{id}/info.0.json")
    Call<Comic> getComic(@Path("id") int comicId);
}

File: app/src/main/java/com/dev/ome/corentin/xkcdapp/XkcdService.kt
package com.dev.ome.corentin.xkcdapp

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface XkcdService {
    @GET("{id}/info.0.json")
    fun getComic(@Path("id") comicId: Int): Call<Comic>
}
